UKHomeOffice/sgmr-service
# SPCR-427 Remove unneeded v1 components

### AC / Description of changes
Removes deprecated v1 components, content and files
Moves v1 components that have been refactored for v2 into v2 structure

### To Test

- You can create an account as a new user
- You can validate your account
- You can log in
- You can onboard as a new user
    - No pleasure crafts saved
    - No people saved
    - No voyages created
    - > you see the onboarding banner & can complete the forms
- You can create a new voyage plan
    - You are shown errors when trying to save without required data
    - You can save and exit & see the saved changes 
    - You can save and continue & see the next page
    - You can access the check your answers page from the manage reports links and see your saved details
- You can submit a voyage
- You can cancel a voyage
- You can view existing voyages
- You can change draft voyages
- You can change submitted voyages
- You cannot change cancelled voyages
- You can create a new pleasure craft
- You can edit an existing pleasure craft
- You can delete a pleasure craft
- You can create a new person
- You can edit an existing person
- You can delete a saved person
- You can edit your account details




### Notes

There are a lot of deleted and moved files in this PR - many of them are connected and to be able to test features still work it is easier to remove them as a single group

---

**Remember to merge to feature branches not `master`.**
